### PR TITLE
Changes to MobileNetV2 example

### PR DESCRIPTION
**Pull request by michaelpoluektov**
- Move function to save quantized model out of ``obtain_and_optimize_mobilenetv2.py``
- Move lion test image out of the source files and into a binary file and header file (using a single binary would require a linker which is probably unnecessarily complex)
- Update .gitignore to include artifacts
- Update ``examples/README.rst`` to mention the MobileNet example

### CODE REVIEW

**Comment by michaelpoluektov** [comment]
Linux test fails because I'm submitting this from a fork

File: examples/app_mobilenetv2/src/support.cpp [hunk 1]
```diff
@@ -3,507 +3,23 @@
...
 
 // The sample input image is initialized at the beginning of the tensor arena.
-// Before we run inference, the input image is copied to the input tensor 
+// Before we run inference, the input image is copied to the input tensor
 // location in the tensor arena.
-// With this optimization, we don't need an extra array to store the input image.
-// Sample input image is of a LION and of size 160x160x3 = 76800 bytes
+// With this optimization, we don't need an extra array to store the input
+// image. Sample input image is of a LION and of size 160x160x3 = 76800 bytes
+#include <lion.h>
```
Comment (ghost) [inline examples/app_mobilenetv2/src/support.cpp]: I think this is confusing. The tensor arena should be declared here itself, not in lion.h. The data can be in lion.h maybe as a macro?

Comment (michaelpoluektov) [reply examples/app_mobilenetv2/src/support.cpp]: Good point, changed it to a macro

**Review by ghost — APPROVED** [review]
+1

**Pull request merged**